docs/overview.md

```diff
@@ -56,10 +54,54 @@ model = MyModel(5)
 model.step()
 ```
 
-You should see agents 0-4, activated in random order. See the [tutorial](tutorials/intro_tutorial) or API documentation for more detail on how to add model functionality.
+You should see agents 1-5, activated in random order. See the [tutorial](tutorials/intro_tutorial) or API documentation for more detail on how to add model functionality.
 
 To bootstrap a new model install mesa and run `mesa startproject`
 
+### AgentSet and model.agents
+Mesa 3.0 introduces the AgentSet class and the `model.agents` attribute, providing powerful tools for managing agents.
```

Comment:
technically not true because agentset has been around for a while. Perhaps change to something like

Mesa 3.0 makes `model.agents` and the `AgentSet` class central in managing and activating agents.

Reply:
```suggestion
Mesa 3.0 makes `model.agents` and the AgentSet class central in managing and activating agents.
```
Good suggestion, agreed!